package.json

```diff
@@ -45,13 +45,14 @@
   "dependencies": {
     "buffer": "^6.0.3",
     "confusables": "^1.1.0",
+    "core-js": "^3.20.2",
     "crypto-browserify": "^3.12.0",
     "idb-keyval": "^6.1.0",
     "mustache": "^4.2.0",
     "post-me": "^0.4.5",
     "punycode": "^2.1.1",
     "randombytes": "^2.1.0",
-    "skynet-js": "^4.0.20-beta",
+    "skynet-js": "^4.0.22-beta",
```

Comment:
This doesn't matter for now because I've linked to a specific branch in `skynet-js` locally. Changing this line wouldn't affect anything, I just have to keep the `skynet-js` branch up-to-date with recent changes. I will change this line once the `skynet-js` branch has been merged and a version change published.